deploy/kubernetes/README.adoc

```diff
@@ -171,50 +178,29 @@ Show the details of the service broker application service again:
 
 ----
 $ kubectl describe service bookstore-broker -n bookstore-broker
-Name:                     bookstore-broker
 Namespace:                bookstore-broker
 Labels:                   run=bookstore-broker
 Annotations:              <none>
 Selector:                 run=bookstore-broker
-Type:                     NodePort
-IP:                       10.107.161.81
+Type:                     LoadBalancer
+IP:                       10.47.247.7
+LoadBalancer Ingress:     35.197.249.208
 Port:                     <unset>  80/TCP
 TargetPort:               8080/TCP
-NodePort:                 <unset>  32248/TCP
-Endpoints:
+NodePort:                 <unset>  30789/TCP
+Endpoints:                10.44.4.8:8080
 Session Affinity:         None
 External Traffic Policy:  Cluster
 Events:                   <none>
 ----
 
-Note the value of the `NodePort` row.
-
-Show a list of Kubernetes pods running in the cluster:
-
-----
-$ kubectl get pods -n bookstore-broker -o=wide
-NAME                               READY     STATUS    IP            NODE
-bookstore-broker-68f88cbbfb-vvmn4   1/1       Running   10.200.37.3   8ce76de4-4cfc-4bcd-b860-043a69cd2402
-----
-
-Note the value in the `NODE` column for the `bookstore-broker` pod.
 
-Show a list of Kubernetes nodes running in the cluster:
-
-----
-$ kubectl get nodes -n bookstore-broker -o=wide
-NAME                                   STATUS    AGE       VERSION   EXTERNAL-IP
-2e45b4e3-1f9a-439e-b830-e051135f9e52   Ready     1d        v1.9.2    192.168.1.236
-8ce76de4-4cfc-4bcd-b860-043a69cd2402   Ready     1d        v1.9.2    192.168.1.235
-cf353f58-bb1a-45cf-8b18-96976f818c5f   Ready     1d        v1.9.2    192.168.1.234
-----
-
-Note the value in the `EXTERNAL-IP` column of the node whose `NAME` matches the node for the `bookstore-broker` pod.
+Note the value in the `LoadBalancer Ingress` column and the `Port` column below.
 
 Construct a URL using the IP address of the node and the port of the service, and use the this URL to access the `/v2/catalog` endpoint of the service broker application:
 
 ----
-$ curl http://192.168.1.235:32248/v2/catalog -u admin:supersecret
+$ curl http://35.197.249.208/v2/catalog -u admin:supersecret
```

Comment:
Will this IP address be valid for all deployments?

Reply:
No but I can't see a way of making an IP deterministic without creating and external DNS record